#!/bin/sh /usr/share/dpatch/dpatch-run
## 02-fix_comparison_warning.dpatch by Roman I Khimov <roman@khimov.ru>
##
## DP: Fixes signed/unsigned comparison warning

@DPATCH@

--- cutecom-0.14.0/qcppdialogimpl.cpp.orig	2006-11-24 21:22:54.669273000 +0300
+++ cutecom-0.14.0/qcppdialogimpl.cpp	2006-11-24 21:23:13.046421500 +0300
@@ -195,7 +195,7 @@ void QCPPDialogImpl::saveSettings()
       historyCount=50;
 
    QStringList saveHist;
-   for (int i=m_oldCmdsLb->count()-historyCount; i<m_oldCmdsLb->count(); i++)
+   for (unsigned int i=m_oldCmdsLb->count()-historyCount; i<m_oldCmdsLb->count(); i++)
       saveHist<< m_oldCmdsLb->text(i);
    settings.writeEntry("/cutecom/History", saveHist);
 }
